script/update-docs.rb

```diff
@@ -384,10 +385,11 @@ def index_doc(filter_tags, doc_list, get_content)
 
       doc_files.each do |entry|
         path, sha = entry
+        txt_path = path.sub(/\.adoc$/, '.txt')
```

Comment:
I wonder if we should put in a hardcoded list with `old_name -> new_name` to only convert the name for existing files, not for docs that are added in the future.

Reply:
I know it's unfashionable in the Git project to aim for consistency, but call me old-fashioned: I do care for consistency. Therefore I like the current form better than to special-case existing files.

Reply:
Not worth the hassle, in my opinion. I think it's only worth it if we have situation which do not work without such map: For example when a file without a `txt` extension was renamed to `.adoc`, but I don't see anything break with current approach.